cmd/zpool/zpool_main.c

```diff
@@ -2389,6 +2392,9 @@ zpool_do_import(int argc, char **argv)
 				mntopts = optarg;
 			}
 			break;
+		case 'p':
+			flags |= ZFS_IMPORT_SCRUB_PAUSE;
+			break;
```

Comment:
Scrubs have always been (silently) paused on readonly pools.  Scrub happens from spa_sync(), and requires modifying the on-disk format (the scn_queue_obj), so it isn't possible to scrub a readonly pool.